NineChronicles.Headless/GraphTypes/ActivationStatusQuery.cs

```diff
@@ -68,6 +69,62 @@ public ActivationStatusQuery(StandaloneContext standaloneContext)
                     }
                 }
             );
+
+            Field<NonNullGraphType<BooleanGraphType>>(
+                name: "addressActivated",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Name = "addressHex"
```

Comment:
This is not `StringGraphType`. How about using `address` directly?
```suggestion
                        Name = "address"
```